app/models/image_variant.rb

```diff
@@ -5,17 +5,12 @@ class ImageVariant
   VARIANTS = {
     small_avatar: 80,
     medium_avatar: 96,
+    card_avatar: 440,
     large_avatar: 960,
-    playlist_avatar: 440,
     playlist_card: 440,
     playlist_cover: 1248,
```

Comment:
I would keep the number of sizes to an absolute minimum, in the end it's going to be faster for users to download a slightly bigger variant once than having to download 4 size variants of the same thing. Especially if they end up downloading the large variant at some point anyway.

Reply:
These are pretty much the current minimum, though we could probably remove the 80px one. The large_avatar and playlist_cover only show on one page and warrant an additional download.